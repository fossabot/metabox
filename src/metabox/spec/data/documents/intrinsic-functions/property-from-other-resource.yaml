Metabox:
  Description: Can resolve property values from the same resource
  Resources:
    MasterResource:
      Type: "metabox::resource"
      Properties:
        PropertyString1: "PropertyString1-Value"
        PropertyString2: "PropertyString2-Value"
    
    ChildResource:
      Type: "metabox::resource"
      Properties:
        # lookup value from other resource
        Value1: "Fn::GetResourceProperty MasterResource.Properties.PropertyString1"
        Value2: "Fn::GetResourceProperty MasterResource.Properties.PropertyString2"
      