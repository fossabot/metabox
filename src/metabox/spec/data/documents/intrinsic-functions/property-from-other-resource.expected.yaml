Metabox:
  Description: Can resolve property values from the same resource
  Resources:
    MasterResource:
      Type: "metabox::resource"
      Properties:
        PropertyString1: "PropertyString1-Value"
        PropertyString2: "PropertyString2-Value"
    
    ChildResource:
      Type: "metabox::resource"
      Properties:
        # lookup value from other resource
        Value1: "PropertyString1-Value"
        Value2: "PropertyString2-Value"